### PR TITLE
Add timezone to Cron pages

### PR DESCRIPTION
**Pull request by 0pc0deFR**
This pull request add system timezone to cron pages.

It's an alternative for issue #2368

### CODE REVIEW

File: web/templates/pages/add_cron.php [hunk 1]
```diff
@@ -24,6 +24,9 @@
 		<div class="form-container form-container-wide">
 			<h1 class="u-mb20"><?= _("Add Cron Job") ?></h1>
 			<?php show_alert_message($_SESSION); ?>
+			<div class="u-mb20">
+				<p><?= _('System timezone : ') .  '<b>' . $v_timezone . '</b>' ?></p>
+			</div>
```
Comment (AlecRust) [inline web/templates/pages/add_cron.php]:
```suggestion
      <p class="u-mb20">
        <?= _('System timezone') ?>: <span class="u-text-bold"><?= $v_timezone ?></span>
      </p>
```

File: web/templates/pages/list_cron.php [hunk 2]
```diff
@@ -8,7 +8,7 @@
 				</a>
 				<?php if ($panel[$user_plain]["CRON_REPORTS"] == "yes") { ?>
 					<a class="button button-secondary" href="/delete/cron/reports/?token=<?= $_SESSION["token"] ?>">
-						<i class="fas fa-toggle-on icon-green"></i><?= _("Disable Notifications") ?>
+						<i class="fas fa-toggle-off icon-green"></i><?= _("Disable Notifications") ?>
```
Comment (AlecRust) [inline web/templates/pages/list_cron.php]: Why have you changed this to `off`?

File: web/templates/pages/list_cron.php [hunk 3]
```diff
@@ -188,10 +190,12 @@ class="units-table-row-action-link data-controls js-confirm-action"
 		<?php } ?>
 	</div>
 
-	<div class="units-table-footer">
+</div>
+
+<footer class="app-footer">
```
Comment (AlecRust) [inline web/templates/pages/list_cron.php]: Why have you added the global app footer in this template? This should already be included by `includes/app_footer.php`.

**Review by jaapmarcus — CHANGES_REQUESTED** [review]
Please revert the changes of the last commit Docuwiki..

**Comment by 0pc0deFR** [comment]
> Please revert the changes of the last commit Docuwiki..

Yes, I'm sorry

**Comment by AlecRust** [comment]
@0pc0deFR you seem to be resolving my comments without addressing them?

**Comment by 0pc0deFR** [comment]
> @0pc0deFR you seem to be resolving my comments without addressing them?

It's modified now.

File: web/templates/pages/add_cron.php (hunk 1 above)
Comment (AlecRust) [reply web/templates/pages/add_cron.php]: @0pc0deFR can you use this in all cases please?

**Comment by AlecRust** [comment]
I've created #4236 for this.

**Pull request closed without merging**